images-dist/modules/kogito-project-versions/module.yaml

```diff
@@ -30,6 +30,8 @@ envs:
     description: Defines the Quarkus Platform groupId to be used by the builder images. Not intended to be changed by end user.
   - name: "QUARKUS_PLATFORM_VERSION"
     value: "3.20.2.SP1-redhat-00003"
+  - name: "QUARKUS_VERSION"
+    value: "3.20.2.redhat-00004"
```

Comment:
Ensure the QUARKUS_VERSION is set to 3.20.2.redhat-00004 when executing the quarkus-mvn-plugin.sh